src/Microsoft.Azure.SignalRService/ServiceHubContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.AspNetCore.SignalR;
```

Comment:
>Microsoft [](start = 6, length = 9)

system usings at the top. You can change your VS settings to have that as default behavior.

Reply:
Ok